research.html

```diff
@@ -0,0 +1,61 @@
+  <blockquote>Interdisciplinary studies could help to increase the coherence of students' learning experience across disciplines by uncovering areas of overlap and gaps in content coverage, and could facilitate an understanding of how to promote the transfer of knowledge from one setting to another. (National Research Council, 2012b, p. 202). </blockquote>
+To help fill this gap and to inform cross-disciplinary discussions on the improvement of undergraduate education, I conducted a series of studies using interview data to explore how students perceive the conceptual relationship between their chemistry and biology courses. These studies were done in the context of a multi-year introductory course transformation initiative at Michigan State University. Funded by the <abbr title="Association for American Universities’">AAU</abbr> <abbr title="Science, Technology, Engineering, and Mathematics">STEM</abbr> Education Initiative, an interdisciplinary effort was underway to transform the introductory biology, chemistry, and physics courses using the lens of three-dimensional learning (Cooper et al., 2015; Laverty et al., 2016; National Research Council, 2012a).</p>
+<ul>
+  <li>Cooper, M. M., Caballero, M. D., Ebert-May, D., Fata-Hartley, C. L., Jardeleza, S. E., Krajcik, J. S., … Underwood, S. M. (2015). Challenge faculty to transform STEM learning. <i>Science</i>, <i>350</i>(6258), 281–282.</li>
+  <li>Laverty, J. T., Underwood, S. M., Matz, R. L., Posey, L. A., Carmel, J. H., Caballero, M. D., … Cooper, M. M. (2016). Characterizing college science assessments: The Three-Dimensional Learning Assessment Protocol. <i>PLoS ONE</i>, <i>11</i>(9), e0162333.</li>
+  <li>National Research Council. (2012a). <i>A framework for K-12 science education: Practices, crosscutting concepts, and core ideas.</i> Washington, DC: National Academies Press.</li>
+  <li>National Research Council. (2012b). <i>Discipline-based education research: Understanding and improving learning in undergraduate science and engineering.</i> National Academies Press.</li>
+</ul>
+<hr />
+<h2>Connecting Structure–Property and Structure–Function Relationships across the Disciplines of Chemistry and Biology: Exploring Student Perceptions</h2>
+<h3>
+  <a href="https://www.lifescied.org/doi/10.1187/cbe.18-01-0004" target="_blank">Kohn, K. P.; Underwood, S. M.; Cooper, M. M. (2018) <i>CBE-Life Sciences Education</i>, <i>17</i>(2), <i>ar33</i>.</a>
+</h3>
+<p>While many university students take science courses in multiple disciplines, little is known about how they perceive common concepts from different disciplinary perspectives. Structure–property and structure–function relationships have long been considered important explanatory concepts in the disciplines of chemistry and biology, respectively. Fourteen university students concurrently enrolled in introductory chemistry and biology courses were interviewed to explore their perceptions regarding:
```

Comment:
Break the paragraphs up. End this paragraph on line 41 and start a new one on line 47.